components/CostsTable.vue

```diff
@@ -102,10 +87,60 @@
       </template>
     </tbody>
   </table>
+  <p class="fw-lighter small mt-auto">
+    *Based on an assumed value of statistical life:
+    click<CIconSvg class="icon help-icon filter-to-primary p-0 ms-1" role="button">
```

Comment:
What's CIconSvg giving us here when we're just going to put an image tag inside it anyway rather than any svg? I think you could get the same behaviour by just putting the class and role attributes on <img> - that seems to be how it gets rendered anyway.

Reply:
You make a good point. I think the only benefit of this is to clue in the developer that "icon" is a CSS class originating from CoreUI. Replacing with img!